scripts/bulk-process.py

```diff
@@ -95,7 +95,7 @@ def get_capa_results(args):
     rules, format, path = args
     logger.info("computing capa results for: %s", path)
     try:
-        extractor = capa.main.get_extractor(path, format, disable_progress=True)
+        extractor = capa.main.get_extractor(path, format, "vivisect", disable_progress=True)

```

Comment:
i'd recommend that we make the "vivisect" string a constant, but its not a huge deal
```suggestion
        extractor = capa.main.get_extractor(path, format, capa.features.extractors.VIVISECT, disable_progress=True)

```

Reply:
though id like to hear thoughts from others

@mr-tz @mike-hunhoff @Ana06

Reply:
even better, an enum
https://docs.python.org/3/library/enum.html

Reply:
let's go Python 3

Reply:
enums were introduced in Python 3.4. So we can't use them until we drop Python2 support.

I have changed by a constant. Although I am using `capa.main.BACKEND_VIV` and not `capa.features.extractors.VIVISECT`. @williballenthin are you proposing defining this constant in a different place (and not in `capa/main.py` as it is done now)?

Reply:
we can use enum34 (https://pypi.org/project/enum34/) for cross version enums

Reply:
> @williballenthin are you proposing defining this constant in a different place 

nope, i forgot about our existing discussion. use the constants we have already!

Reply:
@williballenthin 

> we can use enum34 (pypi.org/project/enum34) for cross version enums

I like to think that we are going to get rid of Python 2 that soon that the effort is not worthwhile 😄